programs/autocrat/src/instructions/finalize_proposal.rs

```diff
@@ -147,38 +152,41 @@ impl FinalizeProposal<'_> {
             .saturating_mul(MAX_BPS.saturating_add(dao.pass_threshold_bps).into())
             / MAX_BPS as u128;
 
-        let (new_proposal_state, new_vault_state) = if pass_market_twap > threshold {
-            (ProposalState::Passed, VaultStatus::Finalized)
+        let (new_proposal_state, payout_numerators) = if pass_market_twap > threshold {
+            (ProposalState::Passed, vec![0, 1])
```

Comment:
ok, i see now that these are the "numerators" that say the proposal 100% passed or 100% failed. On the surface of it, not super intuitive. Don't have an immediate suggestion though.